spline-testing/jmetered-backed-simple/run_jmetered_spline.py

```diff
@@ -87,7 +88,7 @@ def run_docker_compose():
     os.makedirs(f"{root_dir}/{RESULTS_FOLDER_NAME}", exist_ok=True)
     # --exit-code-from = reports exit code from this container
     # AND implies '--abort-on-container-exit' - will 'docker-compose down' after any container has exited
-    subprocess.run("docker-compose up --exit-code-from jmeter")
+    subprocess.run("docker-compose up --exit-code-from jmeter", shell=True)
```

Comment:
Every `subpricess.run()` should propagate errors. What about setting `check=True` attribute for every `run()` call?

From the [doc](https://docs.python.org/3/library/subprocess.html):
> If _check_ is true, and the process exits with a non-zero exit code, a [CalledProcessError](https://docs.python.org/3/library/subprocess.html#subprocess.CalledProcessError) exception will be raised. Attributes of that exception hold the arguments, the exit code, and stdout and stderr if they were captured.

Reply:
Thanks, applied.